clash-ghc/src-ghc/Clash/GHC/Evaluator.hs

```diff
@@ -3211,6 +3244,9 @@ integerLiterals' = typedLiterals' integerLiteral
  where
   integerLiteral x = case x of
     Lit (IntegerLiteral i)      -> Just i
+    DC dc [Left (Literal (IntegerLiteral i))]
```

Comment:
This shouldn’t exist! What is the name of this constructor??? The constructor with tag 1, `S#` (http://hackage.haskell.org/package/integer-gmp-1.0.2.0/docs/GHC-Integer-GMP-Internals.html#t:Integer), takes an Int literal, not an Integer literal. Is GHC giving us malformed Integers??? Or did we create them?

Reply:
Actually, I think this is just a mistake on my part after misreading debugging output and accidentally leaving it there. I'll double-check in the morning.

Reply:
Turns out I indeed misread the logs. I've scrapped ^this^.